src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1891,6 +1891,12 @@ namespace UiaOperationAbstraction
         }
 
         UiaTuple(const UiaTuple&) = default;
+        UiaTuple& operator=(const UiaTuple&) = delete;

```

Comment:
Why is this deleted but the companion method is default? In general, either both are allowed or both are deleted. I think this creates confusion.

Reply:
I am doing this because `UiaTypeBase` is doing similar:
```
        UiaTypeBase(const UiaTypeBase&) = default;
        UiaTypeBase& operator=(const UiaTypeBase&) = delete;
```